src/schema/color.ts

```diff
@@ -6,6 +6,8 @@ export interface ColorBaseInfo {
   name?: string;
   undertone?: Undertone;
   usage?: string;
+
+  location?: number;
```

Comment:
`location` should be a specific property in the 'continuous' case, not for general Color type.

BTW, a location should be in range [0,1]

typescript cannot restrict number to a certain range but JSON schema does. https://json-schema.org/understanding-json-schema/reference/numeric.html#id7

check how vega-lite (e.g.   `"maximum": 360` in `"labelAngle"`) make it via [ts-json-schema-generator](https://github.com/vega/ts-json-schema-generator)